pandas/tests/frame/methods/test_combine_first.py

```diff
@@ -140,9 +140,13 @@ def test_combine_first_mixed_bug(self):
         )
         df2 = DataFrame([[-42.6, np.nan, True], [-5.0, 1.6, False]], index=[1, 2])
 
-        result = df1.combine_first(df2)[2]
+        result1 = df1.combine_first(df2)[2]
+        result2 = df2.combine_first(df1)[2]
+        # this would fail prior to this fix
```

Comment:
can you not comment out code. and remove '# this would fail prior to this fix' which is not a useful comment (except for on the diff), but only we are seeing this

Reply:
Add the comment to point out which case would fail in master, will remove.